vllm/model_executor/models/deepseek_v2.py

```diff
@@ -1275,6 +1275,31 @@ def __init__(self, *, vllm_config: VllmConfig, prefix: str = ""):
                 vllm_config, prefix, topk_indices_buffer=topk_indices_buffer
             ),
             prefix=f"{prefix}.layers",
+            offloader_kwargs=dict(
+                # Extract the MLP submodule - for MoE layers, go deeper to the experts
+                submodule_accessor=lambda layer: (
+                    layer.mlp.experts
+                    if isinstance(layer.mlp, DeepseekV2MoE)
+                    else layer.mlp
+                ),
+                # Specify which parameters to offload
+                whitelist_param_names_creator=lambda module: (
+                    [
+                        # Core MoE expert weights
+                        "w13_weight",
+                        "w2_weight",
+                        # NVFP4 quantization scales (if present)
+                        *(
+                            ["w13_blockscale_swizzled", "w2_blockscale_swizzled"]
+                            if hasattr(module, "w13_blockscale_swizzled")
+                            else []
+                        ),
```

Comment:
Is there harm in offloading "too many" params? I feel like I'd prefer a general "w13" or "w2" match rather than specializing on specific quant methods